docs/api/scheduler.md

```diff
@@ -434,6 +434,76 @@ Reponse
 }
 ```
 
+## Configuration update
+
+Endpoint to override scheduler configuration. 
+
+**URL** : `/ws/v1/config`
+
+**Method** : `PUT`
+
+**Auth required** : NO
+
+### Success response
+
+**Code** : `200 OK`
+
+**Content example**
+
+```yaml
+partitions:
+  -
+    name: default
+    placementrules:
+      - name: tag
+        value: namespace
+        create: true
+    queues:
+      - name: root
+        submitacl: '*'
+        properties:
+          application.sort.policy: stateaware
+checksum: [173,7,83,162,71,12,24,54,61,72,195,227,207,95,85,14,128,101,107,7,112,214,125,208,136,131,212,118,44,191,90,222]
+```
+**Note:** the checksum is the base configuration checksum on top of what we want to make the changes.
+If the provisioned checksum differs from the actual scheduler checksum, the configuration update will fail.
+
+### Failure response
+
+The configuration update can fail due to different reasons such as: 
+- invalid configuration, 
+- outdated base checksum, 
+- failure while saving the configuration to the scheduler. 
```

Comment:
this line is quite confusing... I think this is to outline what could be the failures
this will be hard for users to understand why the saving fails..

Reply:
While saving the scheduler configuration it may fail due to some unexpected errors while updating the partitions, since if the validation passed, it should be OK in most of the cases, but in any case the response message will have the exact error message.
@yangwwei do you want me to remove this line, or how can it be formulated in a more explicit way?

Reply:
When it fails to save the configs, that means something is invalid in the config, so it falls to the `invalid configuration`. I guess we can just remove this line, that might be less confusing if I am a user.

Reply:
removed